quarkchain/cluster/master.py

```diff
@@ -1837,6 +1865,10 @@ def main():
     loop = asyncio.get_event_loop()
     root_state = RootState(env)
     master = MasterServer(env, root_state)
+
+    if env.cluster_config.ENABLE_GRPC_SERVER:
```

Comment:
I think we want to start grpc server AFTER master started. should probably put this part of code further down (like where we started jsonrpc server)